sklearn/impute/_base.py

```diff
@@ -120,6 +120,8 @@ class SimpleImputer(_BaseImputer):
 
     Read more in the :ref:`User Guide <impute>`.
 
+    .. versionadded:: 0.14
```

Comment:
This has only been available as SimpleImputer since 0.20

versionadded helps users understand whether the same tool can be imported in different versions, so the fact that the functionality existed since 0.14 is irrelevant

Reply:
That's clear. 
Maybe we could mention it ? with something like:

```suggestion
    .. versionadded:: 0.20
        Moved from Imputer in sklearn.preprocessing.
```

Reply:
At first, I was not convinced about mentioning this detail. But after seeing a lot of SO question, I think this worth mentioning something.

```
.. versionadded:: 0.20
   `SimpleImputer` replaces the previous `sklearn.preprocessing.Imputer`
   estimator which is now removed.
```